core/src/epicli/data/common/ansible/playbooks/roles/repository/files/prepare/ubuntu-18.04/add-repositories.sh

```diff
@@ -0,0 +1,19 @@
+wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
```

Comment:
Please remove "prepare" directory, there is a new structure, and this file will be placed in repository/files/download-requirements/ubuntu-18.04/add-repositories.sh , see PR #580